Description: Changed gmail.google.com to mail.google.com.
Bug-Debian: http://bugs.debian.org/503740
Author: Alan Woodland <awoodland@debian.org>

--- a/notifier.py
+++ b/notifier.py
@@ -352,8 +352,8 @@
 
 	def gotourl( self, wg=None):
 		print "----------"
-		print "launching browser "+self.options['browserpath']+" http://gmail.google.com"
-		os.system(self.options['browserpath']+" http://gmail.google.com &")	
+		print "launching browser "+self.options['browserpath']+" http://mail.google.com"
+		os.system(self.options['browserpath']+" http://mail.google.com &")
 
 	def show_quota_info( self, event):
 		print "Not available"
